src/app/parser/BaseParser.ts

```diff
@@ -75,7 +75,8 @@ abstract class BaseParser extends BaseElements implements Parser {
 		};
 
 		return Promise.resolve(Promise.props(workflow))
-		.then((a : ParserWorkflowPayload) : Article => ({
```

Comment:
You can change it back. If we do not open "{}" in arrow function - that means we "return" selected result, so no need to use "return" param inside, just get off "{}"

Reply:
PS - ES6